Caboodle/Battery/Battery.android.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Android;
+using Android.Content;
+using Android.OS;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Battery
+    {
+        static bool hasBatteryStatsPermission;
+        static void ValidateBatteryStatsPermission()
+        {
+            if (hasBatteryStatsPermission)
+                return;
+
+            var permission = Manifest.Permission.BatteryStats;
+            if (!Platform.HasPermissionInManifest(permission))
+                throw new PermissionException(permission);
+
+            hasBatteryStatsPermission = true;
+        }
```

Comment:
What happens here if it is in the manifest, but the user has denied permission? Can the user deny permission?

Reply:
This is not a runtime permission so it is okay.